charts/dex/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if and .Values.servicemonitor.enabled }}
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: {{ template "dex.fullname" . }}
+{{- if .Values.servicemonitor.namespace }}
+  namespace: {{ .Values.servicemonitor.namespace }}
+{{- else }}
+  namespace: {{ .Release.Namespace | quote }}
+{{- end }}
+  labels:
+    prometheus: {{ .Values.servicemonitor.prometheusInstance }}
+    app: {{ template "dex.name" . }}
```

Comment:
This should already be included in `dex.labels` template:

https://github.com/dexidp/helm-charts/blob/master/charts/dex/templates/_helpers.tpl#L49

Reply:
Yes, it is. Will remove it.